src/Tribe/Query.php

```diff
@@ -45,6 +47,14 @@ public static function parse_query( $query ) {
 
 		$tec_post_types   = [ TEC::POSTTYPE, Venue::POSTTYPE, Organizer::POSTTYPE ];
 
+		// Bail if we're not dealing with any event or event related queries.
```

Comment:
This will prevent events from being added to the main blogs loop. Could you test this?